docs/sources/setup-grafana/configure-security/configure-authentication/generic-oauth/index.md

```diff
@@ -205,7 +287,7 @@ allowed_organizations =
 
 By default, a refresh token is included in the response for the **Authorization Code Grant**.
 
-## Set up OAuth2 with Centrify
```

Comment:
I'm tempted to remove this section, as Centrify has been taken over by Delinea, so I imagine this is not accurate anymore. But the sign-up process to test it out is super cumbersome, so I haven't had the chance to test it.